_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+## Looking back
+
+Looking back is always easy, but back when we established our procurement strategy, we could only rely on our past experience in the industry, with no guarantees that our strategy relying on purchasing "as late as possible" was going to be effective.
+
+One of the external factors is the USD -> CDN conversion rate, as currency fluctuations directly influence our purchase prices. It probably explains why the cost per TB on an individual disk was higher in September 2016. Our storage needs at that time was only of 3 servers (864 TB), so the impact was fairly limited and the conversion rate started to go back down after that time and is today at the same level it was early 2015 (although still much higher than when the project started in 2014).
+
+<center>
+  <figure style="width: 70%;">
+      <img src="{{site.urlimg}}francois_gerthoffert/collab-procurement/usd-cdn-conversion-rate.png"/>
+      <figcaption>USD-CDN exchange rate evolution</figcaption>
+  </figure>
+</center>
+
+But ultimately, the thing to look at is that today we are purchasing servers with 2.5 times more storage capacity than in 2015, which means less need for physical resources per TB of storage, which plays largely in our favor from a budget point of view.
+
+# Conclusion
+
+So what could be the recommendations and conclusion of this article ?
+
+If you know upfront that your project will be composed of multiple purchases, identify a procurement strategy and setup a contractual framework before starting. Not having to re-negotiate or re-validate items such as warranty, support details, greatly eases the procurement process.
+
+Don't purchase more storage capacity than required. Considering that transferring petabytes of data takes a long time, and the data to be stored might not yet be available, you might have sufficient time for staggered purchases. Instead, identify your requirements and schedule your purchases accordingly. This will provide the project with better value per dollar spent by benefitting of costs decrease and technological improvements.
+
+Watch out for external factors, such as currency exchange rate. Even if they are outside of your control, you might want to try to articulate your strategy around those.
+
+Establish a clear, fair and re-usable evaluation matrix. This will facilitate vendor evaluation and could also be of a great help may a vendor decide to contest an award. This is particularly true in the public sector. Furthermore, in the context of a particular purchase, don't talk to vendors directly before a quote is awarded, let your procurement department handle the communication. This will help prevent disputes coming from vendors. IT is a relatively small ecosystem and it's not rare from vendors to talk to each other or hear rumors and the last thing you want is to have to re-run a quotation process.
```

Comment:
IT is a relatively small ecosystem and it’s not rare for vendors to talk to each other